package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest",
+    "test": "jest -o",
```

Comment:
I would leave this as `test` and add a `test:staged` job or something similar.  Reason being is you will have to update the CI/CD pipeline see: https://github.com/madetech/mca-beacons-webapp/pull/28/checks?check_run_id=1784626760

And running `npm run test` feels more natural to run all the tests rather than having `npm run test:all`